doc/travis-ci.md

```diff
@@ -0,0 +1,80 @@
+# Travis CI
+
+Once you have your tests running and passing in the cloud, it is time to setup airtap to run in a continuous integration server. For this we will use [Travis](https://travis-ci.org/) but any of your favorite CI services will work with airtap.
+
+### 1. getting started
+
+Take a look at the Travis [getting started](http://about.travis-ci.org/docs/user/languages/javascript-with-nodejs/) guide for node.js. Don't worry! Your project doesn't have to be in node.js, we just use that guide to test our javascript.
+
+### 2. enable the Travis webhook
+
+Visit your [Travis profile](https://travis-ci.org/profile) and make sure the repo you want to test is enabled. This is required before moving on to the next steps.
+
+### 3. create a travis.yml config file
+
+Similar to the `.airtap.yml` config file we created for airtap, we need to create a simple file for Travis. In your project directory, open `.travis.yml` with your editor
+
+```yaml
+language: node_js
+node_js:
+  - '0.10'
```

Comment:
Maybe we should tweak this to use more modern node versions.

Reply:
I'll replace 0.10 with 8